contrib/skopeo/template.py

```diff
@@ -0,0 +1,49 @@
+pkgname = "skopeo"
+pkgver = "1.6.0"
+pkgrel = 0
+build_style = "go"
+# for compatibility with Makefile targets
+make_dir = "bin"
+make_build_args = [
+    "-mod",
+    "vendor",
+    "./cmd/skopeo",
+]
+hostmakedepends = [
+    "bash",
```

Comment:
```suggestion
```

not used

Reply:
Sadly not true - their Makefile calls it.

Reply:
it does not call it for docs, install-docs, or install-completions

Reply:
It calls it for the Makefile itself.

Reply:
yeah, and what do you think that does for the build?

(the answer is nothing)

Reply:
> yeah, and what do you think that does for the build?
> 
> (the answer is nothing, because you don't build any binaries with the makefile)

It kills the error message that comes from gmake trying to call bash uselessly.